src/jdk.jlink/share/classes/jdk/tools/jlink/internal/TaskHelper.java

```diff
@@ -30,18 +30,8 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.text.MessageFormat;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Locale;
-import java.util.Map;
+import java.util.*;
```

Comment:
I assume this change is unintentional.  Probably IDE converted the single-class imports to this.